core/src/main/java/io/kestra/core/models/templates/Template.java

```diff
@@ -93,6 +109,23 @@ public Optional<ConstraintViolationException> validateUpdate(Template updated) {
         }
     }
 
+    public String generateSource() {
```

Comment:
this method is not necessary, a simple mapper will generate the source on templates

Reply:
Yes, I setup the object mapper once for all and use it directly.
By the way, I think the same can be done on the flow class, as the `Flow.generateSource()` method is now only used in tests this should be fine to change it the same way I did for templates.